lookout/style/format/features.py

```diff
@@ -11,6 +11,9 @@
 from lookout.core.api.service_analyzer_pb2 import Comment
 from lookout.core.api.service_data_pb2 import File
 
+N_ROLES = len(bblfsh.aliases.DESCRIPTOR.enum_types_by_name["Role"].values_by_number.items())
+ROLES = [bblfsh.role_name(role) for role in range(N_ROLES)]
```

Comment:
Since we merged @zurk's PR, we should use the data which he collected

Reply:
It is right. Only 88 roles are used in JavaScript
https://doc.bblf.sh/languages.html

Reply:
So instead of using a global variable, take it for each language

Reply:
See https://github.com/src-d/style-analyzer/blob/master/lookout/style/format/features.py#L182